packages/ui-test-utils/src/utils/generateA11yTests.tsx

```diff
@@ -45,21 +41,24 @@ type ExampleSection = {
   }[]
 }
 
-const renderExample = ({
-  Component,
-  componentProps,
-  key
-}: ComponentExample) => <Component key={key} {...componentProps} />
+//const renderExample = ({
+//  Component,
+//  componentProps,
+//  key
+//}: ComponentExample) => <Component key={key} {...componentProps} />
 
 // this is coming from generateComponentExamples.js in __examples__
 // TODO This is broken. It was using the examples-loader to preprocess the data.
 export function generateA11yTests(
```

Comment:
This is technically a breaking change, but since nobody should be using this utility anyways it is okay in this case to merge it.